test/parallel/test-http-mutable-headers.js

```diff
@@ -50,18 +50,38 @@ const s = http.createServer(common.mustCall((req, res) => {
       assert.deepStrictEqual(res.hasHeader('Connection'), false);
       assert.deepStrictEqual(res.getHeader('Connection'), undefined);
 
-      assert.throws(() => {
-        res.setHeader();
-      }, /^TypeError: Header name must be a valid HTTP Token \["undefined"\]$/);
-      assert.throws(() => {
-        res.setHeader('someHeader');
-      }, /^Error: "value" required in setHeader\("someHeader", value\)$/);
-      assert.throws(() => {
-        res.getHeader();
-      }, /^TypeError: "name" argument must be a string$/);
-      assert.throws(() => {
-        res.removeHeader();
-      }, /^TypeError: "name" argument must be a string$/);
+      common.expectsError(
+        () => res.setHeader(),
+        {
+          code: 'ERR_INVALID_HTTP_TOKEN',
```

Comment:
@starkwang how much work will it to add the given value (in addition to the token name)?